test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -97,12 +98,12 @@ waitForNodes "${pids[@]}"
 echo "Done!"
 
 echo "Copying Antrea deployment YAML to every node..."
-scp -F ssh-config $ANTREA_YML $ANTREA_IPSEC_YML k8s-node-master:~/ &
+scp -F ssh-config $ANTREA_YML $ANTREA_IPSEC_YML $FLOW_AGGREGATOR_YML k8s-node-master:~/ &
```

Comment:
I am guessing this is for your local test runs and not CI. 
We should probably automate the workflow of enabling flow exporter, deploying Antrea, and deploying Flow Aggregator on the vagrant setup for dev/demo purposes. This can be considered as a future TODO.

Reply:
If its not for your CI, please remove this. https://github.com/vmware-tanzu/antrea/pull/1646 can handle this for dev automated workflow.

Reply:
Thanks. removed.